Gaijinizer patch/Map369.txt

```diff
@@ -268,667 +320,856 @@ You don't have to be so concerned...#MaybeWrong?
 > END STRING
 
 > BEGIN STRING
 でも…………今だから思うんだけど
 僕はきっと、『アイドル』になりたかったわけじゃない。
 
 『ニーナちゃん』になりたかったんだと思う……
-> CONTEXT: Map369/events/5/pages/1/176/Dialogue/enemy26 < UNTRANSLATED
+> CONTEXT: Map369/events/5/pages/1/176/Dialogue/enemy26
+But... thinking about it now...
 
+It's not that I wanted to become an idol like her,
+I think I actually just wanted to be her.#Ron
 > END STRING
 
 > BEGIN STRING
 カメラの向こうで、大勢が見てるって分かってるのに
 あんなに笑顔で、明るく振る舞って…『ホンモノ』を見せる
 
 まさしく、僕が求める『理想の姿』だったから…
-> CONTEXT: Map369/events/5/pages/1/181/Dialogue/enemy26 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/181/Dialogue/enemy26
+Facing the cameras, knowing I'm being watched by
+huge numbers of people, yet still bright and with
+a huge smile on my face, showing them all the real
+me. That was the "ideal" I was chasing.#Ron
 > END STRING
 
 > BEGIN STRING
 だから、僕もニーナちゃんみたいに自信を付けたくて
 カメラを意識した、番組を進行するような感じで
 
 毎日毎日、バーゲストと一緒に特訓してたんだ！
-> CONTEXT: Map369/events/5/pages/1/186/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/186/Dialogue/enemy25
+That's why I had that Barghest help me train
+day after day to build the confidence to be
+conscious of the camera and properly progress
+a show like Nina-chan.#Ron
 > END STRING
 
 > BEGIN STRING
 ただまぁ、しょせんはテレビの真似事だからね……
 
 番組の冒頭説明くらいしか、まだできなかったけど…
-> CONTEXT: Map369/events/5/pages/1/191/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/191/Dialogue/enemy25
+But all that practice was never anything more
+than make-believe. And I couldn't do anything
+more than pretend to do a show's opening spiel.#Ron
 > END STRING
 
 > BEGIN STRING
 ああ………
 
 だから、番組の始まった冒頭以外は
 全然カメラを意識してない素振りだったのか………
-> CONTEXT: Map369/events/5/pages/1/195/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/195/Dialogue/kazuya
+Aaah, is that why apart from your opening bit,
+you didn't seem to have any awareness of the
+camera at all?#Ron
 > END STRING
 
 > BEGIN STRING
 それじゃあ、僕をあの場所に呼んだのも
 
 最初から『特訓』の一環だったの？
-> CONTEXT: Map369/events/5/pages/1/200/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/200/Dialogue/kazuya
+And when we first met after I was summoned, was
+that your training I saw?#Ron
 > END STRING
 
 > BEGIN STRING
 うーん……僕としては、
 もうちょっと自信をつけた後がよかったんだけどね…
 
 あれは実際、ヘカーテ様がキミを呼んだわけだから……
-> CONTEXT: Map369/events/5/pages/1/204/Dialogue/enemy24 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/204/Dialogue/enemy24
+Mmmm, I would've liked to have built a little
+more confidence before you came along. You see,
+it was actually Hecate-sama who summoned you, not
+me.#Ron
 > END STRING
 
 > BEGIN STRING
 ていうか、『そういう作戦があるから』とは聞かされても
 あんなに突然呼ばれるなんて思ってなかったんだよ……
 
 だからあの時も、悠長にリハーサルやってたんだよね……
-> CONTEXT: Map369/events/5/pages/1/209/Dialogue/enemy24 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/209/Dialogue/enemy24
+And also, though I'd been told about the plan,
+I didn't think you'd be summoned so quickly.
+Otherwise I wouldn't have been rehearsing so
+leisurely.#Ron
 > END STRING
```

Comment:
Minor rewording for flow:
```
Also,  even though I'd been told about the plan,
I didn't think you'd be summoned so quickly.
Otherwise I wouldn't have been taking time
rehearsing.
```